includes/Elements/Business_Reviews.php

```diff
@@ -752,6 +752,46 @@ protected function register_controls() {
 
 		$this->end_controls_section();
 
+		/**
+		 * Accessibilty Controller
+		 */
+		$this->start_controls_section(
+			'eael_section_business_reviews_accessibilty',
+			[
+				'label' => esc_html__( 'Accessibilty', 'essential-addons-for-elementor-lite' ),
+			]
+		);
+
+		$this->add_control(
+			'eael_business_reviews_enable_accessibilty',
+			[
+				'label'        => __( 'Enable Accessibilty', 'essential-addons-for-elementor-lite' ),
+				'type'         => Controls_Manager::SWITCHER,
+				'label_on'     => __( 'Yes', 'essential-addons-for-elementor-lite' ),
+				'label_off'    => __( 'No', 'essential-addons-for-elementor-lite' ),
+				'return_value' => 'yes',
+				'default'      => 'no',
+			]
+		);
+
+		$this->add_control(
+			'eael_business_reviews_link_in_same_tab',
+			[
+				'label'        => __( 'Open in same window', 'essential-addons-for-elementor-lite' ),
+				'type'         => Controls_Manager::SWITCHER,
+				'label_on'     => __( 'Yes', 'essential-addons-for-elementor-lite' ),
+				'label_off'    => __( 'No', 'essential-addons-for-elementor-lite' ),
+				'return_value' => 'yes',
+				'default'      => 'no',
+				'description'  => __( 'Recommended to open link in same tab instead on a new tab', 'essential-addons-for-elementor-lite' ),
```

Comment:
Update with this text:
`Recommended to open link in the same tab instead of a new tab`